### PR TITLE
Add search to multipass docs

### LINKED ISSUE #148: Add search to docs

### PR DESCRIPTION
**Pull request by carkod**
## QA

- Go to https://multipass-run-149.demos.haus/docs and search something like "application", which usually yields a lot of results
- Make sure the next button works and you can also search in the results page

## Issues
Fixes #148

### CODE REVIEW

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some comments in the code.

File: templates/docs/search.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+              <li class="p-list__item is-ticked">Adding alternative words or phrases</li>
+              <li class="p-list__item is-ticked">Using individual words instead of phrases</li>
+              <li class="p-list__item is-ticked">Trying a different spelling</li>
+            </ul>
+          </div>
+          <div class="col-6">
+            <h3>Still no luck?</h3>
+            <ul class="p-list">
+              <li class="p-list__item is-ticked"><a href="/">Visit the MicroK8s homepage</a></li>
+              <li class="p-list__item is-ticked"><a href="https://discuss.kubernetes.io/c/general-discussions/microk8s/26">Visit the forum</a></li>
```
[the remark this answers is not in the document]
Comment (carkod) [reply templates/docs/search.html]: Done Thanks.

File: templates/docs/search.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      {% endif %}
+    </div>
+  </div>
+
+  {# search form #}
+  <div class="p-strip--light is-shallow">
+    <div class="u-fixed-width">
+      <form class="p-search-box" action="/docs/search">
+        <label for="search-input" class="u-off-screen">Search</label>
+        <input class="p-search-box__input" name="q" id="search-input" type="search" {% if query %}value="{{ query }}"{% endif %} placeholder="e.g. juju" />
```
[the remark this answers is not in the document]
Comment (carkod) [reply templates/docs/search.html]: Done.

**Comment by carkod** [comment]
@anthonydillon sorry I think I put the wrong label, I still need a code review, thanks.

**Review by anthonydillon — APPROVED** [review]
LGTM :+1:

**Pull request merged**